atomkraft/utils/project.py

```diff
@@ -5,7 +5,10 @@
 def project_root():
     cwd = Path(os.getcwd())
     while cwd != cwd.parent:
-        if (cwd / "pyproject.toml").exists():
+        if (
+            (cwd / "pyproject.toml").exists()
+            and (cwd / "atomkraft.toml").exists()
+            and (cwd / ".atomkraft" / "config.toml").exists()
```

Comment:
Project root is the nearest parent with `./pyproject.toml`, `./atomkraft.toml`, `./.atomkraft/config.toml` present.

Reply:
looks good